### PR TITLE
feat: Feature/scrape traffic index

### LINKED ISSUE #8: Scrape traffic-index
Scrape https://www.tomtom.com/en_gb/traffic-index/ranking/ ,use data for where.py script

### PR DESCRIPTION
**Pull request by HAKSOAT**
Resolve #8

### CODE REVIEW

**Comment by serhii73** [comment]
@HAKSOAT Thank you for your code, are you can ti fix all problems what show Travis?

**Comment by serhii73** [comment]
@HAKSOAT Also, please use these rules for writing GIT commits:
```
1. Separate subject from a body with a blank line
2. Limit the subject line to 50 characters
3. Capitalize the subject line
4. Do not end the subject line with a period
5. Use the imperative mood in the subject line
6. Wrap the body at 72 characters
7. Use the body to explain what and why vs. how    
```

**Comment by HAKSOAT** [comment]
> @HAKSOAT Also, please use these rules for writing GIT commits:
> 
> ```
> 1. Separate subject from a body with a blank line
> 2. Limit the subject line to 50 characters
> 3. Capitalize the subject line
> 4. Do not end the subject line with a period
> 5. Use the imperative mood in the subject line
> 6. Wrap the body at 72 characters
> 7. Use the body to explain what and why vs. how    
> ```

Noted.

File: traffic_index/traffic_index/spiders/traffic.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 import scrapy
```
Comment (serhii73) [inline traffic_index/traffic_index/spiders/traffic.py]: @HAKSOAT Please, use `from scrapy import Spider`

File: traffic_index/traffic_index/spiders/traffic.py [hunk 2]
```diff
@@ -8,4 +10,16 @@ class TrafficSpider(scrapy.Spider):
     start_urls = ["https://www.tomtom.com/en_gb/traffic-index/ranking/"]
 
     def parse(self, response):
-        pass
+        xpath_selector = "//*[@id='RankingPage-table']//table//tr/td[{}]"
+        world_ranks = response.xpath(xpath_selector.format(1)).getall()
```
Comment (serhii73) [inline traffic_index/traffic_index/spiders/traffic.py]: @HAKSOAT Please, use [f string ](https://www.python.org/dev/peps/pep-0498/)

File: traffic_index/traffic_index/spiders/traffic.py [hunk 3]
```diff
@@ -8,4 +10,16 @@ class TrafficSpider(scrapy.Spider):
     start_urls = ["https://www.tomtom.com/en_gb/traffic-index/ranking/"]
 
     def parse(self, response):
```
Comment (serhii73) [inline traffic_index/traffic_index/spiders/traffic.py]: @HAKSOAT Can you please add documentation for this class and function?

File: traffic_index/traffic_index/spiders/traffic.py (hunk 1 above)
Comment (serhii73) [reply traffic_index/traffic_index/spiders/traffic.py]: Can you please add documentation for this module?

File: traffic_index/traffic_index/spiders/traffic.py [hunk 4]
```diff
@@ -8,4 +10,16 @@ class TrafficSpider(scrapy.Spider):
...
-        pass
+        xpath_selector = "//*[@id='RankingPage-table']//table//tr/td[{}]"
```
Comment (serhii73) [inline traffic_index/traffic_index/spiders/traffic.py]: We can simplify XPath like `"//div[@id='RankingPage-table']//td[{}]//text()"`
What do you think about rename `xpath_selector` or add such comments, where will understand what the variable means?
`# Base XPath for extract need values`

**Comment by serhii73** [comment]
@HAKSOAT Thank you very much.
I merge your code to master.
Have a nice day, Habeeb.

**Pull request merged**